### PR TITLE
Start working with Flask, adapted fight logic for Flask, changed Character creation.

### PR DESCRIPTION
**Pull request by ArtemGolovatyuk**

### CODE REVIEW

File: characterCreation.py [hunk 1]
```diff
@@ -1,21 +1,15 @@
...
-        return f"{self.name}: критическая атака {self.critical_attack}, здоровье {self.health}, броня {self.armor}, атака {self.attack}, удача {self.luck}"
-
-# Создание персонажей
-character1 = Character("Персонаж 1")
-character2 = Character("Персонаж 2")
-
-# Вывод информации о персонажах
-print(character1)
-print(character2)
+        return f"{self.name}: критическая атака {self.critical_attack}, здоровье {self.health}, броня {self.armor}, атака {self.attack}, удача {self.luck}"
```
Comment (tetyaZyna) [inline characterCreation.py:15]: Code does not comply with PEP 8

File: FlaksMain.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    if character1.health <= 0 and character2.health <= 0:
+        return "Ничья"
+    elif character1.health <= 0:
+        return "Персонаж 2"
+    elif character2.health <= 0:
+        return "Персонаж 1"
+
+
+if __name__ == "__main__":
+    app.run(debug=True)
```
Comment (tetyaZyna) [inline FlaksMain.py:49]: Code does not comply with PEP 8

**Review by MaksGorobets — APPROVED** [review]
Looks good. 🤡🎪

File: characterCreation.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        self.attack = random.randint(1, 20)
+        
+    def __str__(self):
+        return f"{self.name}: критическая атака {self.critical_attack}, здоровье {self.health}, броня {self.armor}, атака {self.attack}, удача {self.luck}"
+    
+
+character1 = Character()
+character2 = Character()
+    
+
```
Comment (tetyaZyna) [inline characterCreation.py:47]: Pep 8

File: FlaksMain.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        if character2.health <= 0:
+            character1.alive = False
+        if Playeryou.health <= 0:
+            return render_template('winner.html')
+
+    return render_template('index.html', character1=character1, character2=character2, Playeryou=Playeryou)
+
+
+if __name__ == "__main__":
+    app.run(debug=True)
```
Comment (tetyaZyna) [inline FlaksMain.py:40]: pep 8

**Review by tetyaZyna — APPROVED** [review]

**Pull request merged**